modules/ai/module.php

```diff
@@ -211,8 +220,6 @@ public function ajax_ai_get_custom_css( $data ) {
 	}
 
 	public function ajax_ai_set_get_started( $data ) {
-		$this->verify_permissions( $data['editor_post_id'] );
```

Comment:
Why removed?

Reply:
When trying to connect in the admin, the API has returned a "document not found" - @KingYes  said that we need to remove this line in order to fix it.

Reply:
@Nevoss I asked it. It's okay for now.